test/edu-benefits/edu-apply-wizard.e2e.spec.js

```diff
@@ -0,0 +1,123 @@
+      .waitForElementVisible('#apply-now-button', Timeouts.normal)
+      .expect.element('#apply-now-button').to.have.css('left').equals('auto');
+
+    client
+      .expect.element('#apply-now-button').to.have.attribute('href').which.contains('/education/apply-for-education-benefits/application/5490/introduction');
+
+    // Select non-dependent
+    client
+      .click('#create-non-dependent')
+      .expect.element('#apply-now-button').to.have.css('left').equals('-9999px');
+
+    client
+      .waitForElementVisible('[data-question="create-transfer"]', Timeouts.normal)
+      .expect.element('[data-question="create-transfer"]').to.have.css('left').equals('auto');
+
+    // Select transfer
+    client
+      .click('#create-transfer')
+      .waitForElementVisible('#apply-now-button', Timeouts.normal)
+      .expect.element('#apply-now-button').to.have.css('left').equals('auto');
+
+    client
+      .expect.element('#apply-now-button').to.have.attribute('href').which.contains('/education/apply-for-education-benefits/application/1990e/introduction');
+
+    // Select non-transfer
+    client
+      .click('#create-non-transfer')
+      .waitForElementVisible('a#apply-now-button', Timeouts.normal)
+      .expect.element('#apply-now-button').to.have.css('left').equals('auto');
+
+    client
+      .expect.element('#transfer-warning').to.have.css('left').equals('auto');
+
+    client
+      .expect.element('#apply-now-button').to.have.attribute('href').which.contains('/education/apply-for-education-benefits/application/1990e/introduction');
+
+    // Update an existing application
+    client
+      .click('#existing-application')
+      .expect.element('#apply-now-button').to.have.css('left').equals('-9999px');
+
+    client
+      .waitForElementVisible('div[data-question="update"]', Timeouts.normal)
+      .expect.element('div[data-question="update"]').to.have.css('left').equals('auto');
+
+    // Select dependent
+    client
+      .click('#update-dependent')
+      .waitForElementVisible('#apply-now-button', Timeouts.normal)
+      .expect.element('#apply-now-button').to.have.css('left').equals('auto');
+
+    client
+      .expect.element('#transfer-warning').to.have.css('left').equals('-9999px');
+
+    client
+      .expect.element('#apply-now-button').to.have.attribute('href').which.contains('/education/apply-for-education-benefits/application/5495/introduction');
+
+    // Select non-dependent
+    client
+      .click('#update-non-dependent')
+      .waitForElementVisible('#apply-now-button', Timeouts.normal)
+      .expect.element('#apply-now-button').to.have.css('left').equals('auto');
+
+    client
+      .expect.element('#apply-now-button').to.have.attribute('href').which.contains('/education/apply-for-education-benefits/application/1995/introduction');
+
+    // Navigate to application
+    client
+      .click('#apply-now-button');
+    E2eHelpers.expectNavigateAwayFrom(client, '/education/apply-wizard/');
```

Comment:
My guess is `/education/apply-wizard/` vs `/education/apply-wizard`, but :man_shrugging: